<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
<suite thread-count="1" guice-stage="DEVELOPMENT" name="Failed suite [SFDC Test Automation Test Suite]" parallel="methods">
  <listeners>
    <listener class-name="com.qa.EventListener.TestListener"/>
    <listener class-name="com.qa.EventListener.FailureRetryListener"/>
  </listeners>
  <test name="Log a Call Test Scenarios(failed)" parallel="methods">
    <classes>
      <class name="com.crm.qa.testcases.LogActivityTest">
        <methods>
          <include name="tearDown"/>
          <include name="setUp"/>
          <include name="reached_Schedule_ToDo" invocation-numbers="1"/>
        </methods>
      </class> <!-- com.crm.qa.testcases.LogActivityTest -->
    </classes>
  </test> <!-- Log a Call Test Scenarios(failed) -->
  <test name="Branch Opportunity Test Scenarios(failed)" parallel="methods">
    <classes>
      <class name="com.crm.qa.testcases.BranchOpportunityTest">
        <methods>
          <include name="scheduleMeetingviaNextAction" invocation-numbers="1 1"/>
          <include name="setUp"/>
          <include name="tearDown"/>
        </methods>
      </class> <!-- com.crm.qa.testcases.BranchOpportunityTest -->
    </classes>
  </test> <!-- Branch Opportunity Test Scenarios(failed) -->
</suite> <!-- Failed suite [SFDC Test Automation Test Suite] -->
